project/build.sbt

```diff
@@ -7,6 +7,7 @@ lazy val root = project
     addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.9.3"),
     addSbtPlugin("ch.epfl.scala" % "sbt-release-early" % "1.1.0"),
     addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.11"),
+    // addSbtPlugin("ch.epfl.scala" % "sbt-profiling" % "8cbd55b0+20171004-1555"),
```

Comment:
This can be uncommented once this PR is merged and can be used to get statistics from warmed up compilers in our integration tests.